podcasts/DiscoverCollectionViewController.swift

```diff
@@ -0,0 +1,69 @@
+class DiscoverCollectionViewController: UIViewController, UICollectionViewDelegate {
+
+    enum Cell {
+        case list
+
+        var reuseIdentifier: String {
+            switch self {
+            case .list:
+                "listCell"
+            }
+        }
+    }
+
+    private var collectionView: UICollectionView!
+
+    private let coordinator: DiscoverCoordinator
+
+    init(coordinator: DiscoverCoordinator) {
+        self.coordinator = coordinator
+
+        super.init(nibName: nil, bundle: nil)
+    }
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        collectionView = UICollectionView(frame: view.bounds, collectionViewLayout: collectionViewLayout())
+        collectionView.backgroundColor = .white
+        collectionView.dataSource = self
+        collectionView.delegate = self
+
+        collectionView.register(UICollectionViewListCell.self, forCellWithReuseIdentifier: Cell.list.reuseIdentifier)
+
+        view.addSubview(collectionView)
+    }
+
+    private func collectionViewLayout() -> UICollectionViewLayout {
+        return UICollectionViewCompositionalLayout { (sectionIndex, layoutEnvironment) -> NSCollectionLayoutSection? in
+            let itemSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1.0), heightDimension: .absolute(100))
+            let item = NSCollectionLayoutItem(layoutSize: itemSize)
+            let groupSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1.0), heightDimension: .absolute(100))
+            let group = NSCollectionLayoutGroup.vertical(layoutSize: groupSize, subitems: [item])
+            let section = NSCollectionLayoutSection(group: group)
+            return section
+        }
+    }
+}
+
+// MARK: - UICollectionViewDataSource
+
+extension DiscoverCollectionViewController: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return 1
```

Comment:
I believe this is temporary, but if the number won't change you can omit the whole method

Reply:
Definitely temporary since I'll switch to a diffable data source but isn't this method required? I think it's `numberOfSections` which is optional.

Reply:
Nevermind, my bad!